lib/resolve/secrets.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const fs = require('fs'),
+    path = require('path'),
+    SECRETS_DIR = '/run/secrets',
```

Comment:
It would be nice to make it more generic. Make a parameter out of this value.

Reply:
Is not possible, docker always search for secrets in this path.